scheduler/scheduler-api/src/main/java/org/ow2/proactive/scheduler/common/Scheduler.java

```diff
@@ -1737,4 +1737,26 @@ TaskStatesPage getTaskPaginated(String jobId, String statusFilter, int offset, i
      */
     Map<String, Map<String, Boolean>> checkJobsPermissionMethods(List<String> jobIds, List<String> methods)
             throws NotConnectedException, UnknownJobException;
+
+    /**
+     * Add an external endpoint url to the endpoint list of the considered job
+     * @param jobId id of the job
+     * @param externalEndpointUrl the external endpoint url to remove from the job endpoint list
+     * @throws NotConnectedException if you are not authenticated.
+     * @throws PermissionException if you can't access to this particular job.
```

Comment:
if you can't access this particular job

Reply:
I will update all the occurrences in this file